server/database/seed.js

```diff
@@ -8,9 +8,9 @@ const file = path.join(__dirname, 'sampleData5.csv');
 const writeReviews = fs.createWriteStream(file, {flags: 'a'});
 
 const seed = (storeStart, storeEnd) => {
-  for (let s = storeStart; s < storeEnd; s++) {                  // 10000 stores
-    for (let d = 0; d < 100; d++) {                              // 100 items
-      for (let c = 0; c < Math.ceil(Math.random() * 11); c++) {  // 5 reviews per item === 5M Rev per Round
+  for (let s = storeStart; s < storeEnd; s++) {
+    for (let d = 0; d < 100; d++) {
```

Comment:
I would avoid putting hard coded numbers in this like 100, you should think about putting it into a variable called numItems or something like that at the top, and referencing that instead. Same applies to other hard coded numbers in this file.

Reply:
Very true, for clarity sake yeah?